### ISSUE TITLE
All disallowed moderator UI actions should be grayed-out

### ISSUE THREAD

**Issue by staltz** [ssb-ngi-pointer/go-ssb-room#150]
Also, non-clickable (UI cursor is the default arrow, and clicking does not lead to another page).

**Comment by cryptix** [comment]
Depends on #157

**Comment by staltz** [comment]
Duplicate of #138
